apollo-router/src/plugins/rhai.rs

```diff
@@ -996,6 +1072,8 @@ impl Rhai {
             ast: self.ast.clone(),
         };
         let mut guard = scope.lock().unwrap();
+        // TODO: LOOKS LIKE WE'VE NEVER BEEN HANDLING THROWN ERRORS HERE. WHEN FINISHED UPDATING
```

Comment:
Is this to be tackled now?

Reply:
No. That's one for future us. We can't handle thrown errors there, since this is the code that builds up the pipeline, so we can't do anything but return a service (i.e.: there is no error path). We'd need a change to the way we build up the pipeline for us to do anything about it. I'll change the comment to make it more clear.

Reply:
done in #aeaa65b3